nautobot/core/views/generic.py

```diff
@@ -219,7 +219,9 @@ def validate_action_buttons(self, request):
         always_valid_actions = ("export",)
         valid_actions = []
         invalid_actions = []
-
+        # added check for whether the action_buttons exist because of issue #2107
+        if self.action_buttons is None:
+            self.actions_buttons = ("export",)
```

Comment:
1. typo `action_buttons` versus `actions_buttons` - was this tested?
2. IMHO this function should fix up a `None` `action_buttons` to `[]`, not `("export",)`

Reply:
1. Sorry for the oversight, now this is tested and works.
2. Does this mean we dont consider "export" as an always valid action?

Reply:
> * typo `action_buttons` versus `actions_buttons` - was this tested?
> * IMHO this function should fix up a `None` `action_buttons` to `[]`, not `("export",)`

🤦

Reply:
I know I cant believe I did that🫠

Reply:
`export` is always valid if it's specified, but not all views may want to support `export` - an empty list of action buttons is valid.